src/usr/local/containerbase/tools/v2/rye.sh

```diff
@@ -0,0 +1,65 @@
+#!/bin/bash
+
+function install_tool () {
+  local versioned_tool_path
+  local arch
+  local name
+  local version
+  local platform
+  local repo
+
+  arch=${ARCHITECTURE}
+  name=$TOOL_NAME
+  version=$TOOL_VERSION
+  platform=$(uname -s)
+  repo="astral-sh/$name"
+
+  if [[ $platform == "Linux" ]]; then
+    platform="linux"
+  fi
+
+  if [[ $arch == armv8* ]] || [[ $arch == arm64* ]] || [[ $arch == aarch64* ]]; then
```

Comment:
please check other installers and simplify, architecture can only have 2 values.